packages/components/src/Menu/Menu.module.css

```diff
@@ -76,8 +76,8 @@
   gap: var(--menu-space);
   align-items: center;
   transition:
-    background-color var(--timing-base) ease-in,
-    box-shadow var(--timing-base) ease-in;
+    background-color var(--timing-base) ease-out,
+    box-shadow var(--timing-base) ease-out;
```

Comment:
This was the only component where we were using `ease-in` for transition and I referenced it when I wrote the guide 😭